src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -805,6 +806,12 @@ private void checkTransferRefund(HyperwalletTransferRefund actual, HyperwalletTr
         assertThat(actual.getNotes(), is(expected.getNotes()));
         assertThat(actual.getMemo(), is(expected.getMemo()));
         assertThat(actual.getCreatedOn(), is(expected.getCreatedOn()));
+        ForeignExchange foreignExchange = actual.getForeignExchanges().get(0);
```

Comment:
i think we should have a separate test for FX. aka, one test with and one test without the foreign exchange. same for the rest of the tests.

Reply:
Created additional tests without FX. Thanks.